### DISCUSSION TITLE
CSS, and Images break after connecting custom domain

### DISCUSSION THREAD

**Discussion by DonovanJTapunha** [community/community#21575] [Other Feature Feedback, Questions, & Ideas]
<p>I built a portfolio using github pages, under the default github url everything worked perfectly.<br>
after connecting a custom domain, only the index page loads but the css and images seem to be disabled.</p>
<p>the site: <a href="https://www.donovanjesse.design/" rel="noopener nofollow ugc">www.donovanjesse.design </a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/3/73dbc7349f3191de76885f34d734097fab41e86f.png" data-download-href="/uploads/short-url/gwVMsvnzO3e3kMfBVFjsWiqWelh.png?dl=1" title="Screen Shot 2021-06-08 at 4.48.04 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/71228877/181065685-7c500389-d314-4f6b-b4c1-fceecf66a0c3.png" alt="Screen Shot 2021-06-08 at 4.48.04 PM" data-base62-sha1="gwVMsvnzO3e3kMfBVFjsWiqWelh" width="517" height="270" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/3/73dbc7349f3191de76885f34d734097fab41e86f_2_517x270.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/3/73dbc7349f3191de76885f34d734097fab41e86f_2_775x405.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/3/73dbc7349f3191de76885f34d734097fab41e86f_2_1034x540.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/3/73dbc7349f3191de76885f34d734097fab41e86f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-06-08 at 4.48.04 PM</span><span class="informations">3554×1860 514 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment, marked as answer]
<p>It looks like this is happening because your asset links still have the repository name hardcoded into the links:</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" href="../Portfolio-Website-1.0/css/style.css"&gt;
</code></pre>
<p>You’ll need to remove <code>/Portfolio-Website-1.0</code> from any asset links, like this:</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" href="../css/style.css"&gt;
</code></pre>
<p>Once you’ve done that for any links to scripts, styles, or images then it should start working!</p>

**Comment by DonovanJTapunha** [comment]
<aside class="quote group-github-employees" data-username="thomasshaped" data-post="2" data-topic="184166">
<div class="title">
<div class="quote-controls"></div>
 thomasshaped:</div>
<blockquote>
<p>scripts, styles, or images then it sh</p>
</blockquote>
</aside>
<p>Amazing, it worked. Thank you!</p>
